### PR TITLE
Get udp port when bind 0

### PR DESCRIPTION
**Pull request by gcxfd**
fix https://github.com/vlang/v/issues/13512

### CODE REVIEW

File: vlib/v/checker/checker.v [hunk 1]
```diff
@@ -24,7 +24,7 @@ const stmt_level_cutoff_limit = 40
 
 const iface_level_cutoff_limit = 100
 
-const (
+pub const (
```
Comment (spytheman) [inline vlib/v/checker/checker.v:27]: that is unrelated to the topic of the PR

Comment (gcxfd) [reply vlib/v/checker/checker.v:27]: https://github.com/vlang/v/commit/67e33bf9bc6591ceb1774bbc3165764bef89b231 this is merged
